src/govuk/components/all.test.js

```diff
@@ -1,51 +0,0 @@
-const { join } = require('path')
-const { fetch } = require('undici')
-const { WebSocket } = require('ws')
```

Comment:
Can we now remove `ws` from our dependencies? (Was introduced in 496cc386817e8360420bf9b590e08bbecabb4e60)

Reply:
Very nice, yep let's do that 🎉